plugins/modules/win_package.py

```diff
@@ -256,6 +256,34 @@
     state: present
     log_path: D:\logs\vcredist_x64-exe-{{lookup('pipe', 'date +%Y%m%dT%H%M%S')}}.log
 
+- name: Install Application from msi with multiple properties for installer
+  ansible.windows.win_package:
+    path: C:\temp\Application.msi
+    state: present
+    arguments: >-
+      /quiet
+      /norestart
+      /log
+      C:\application_server.log
```

Comment:
```suggestion
```

These 4 arguments are already specified when using an MSI in `win_package` and should not be explicitly set.